src/views/address/confirm.html

```diff
@@ -32,7 +32,10 @@ <h1 id="header">{{translate("pages.address-confirm.title")}}</h1>
             items: [
               {
                 href: "/address/edit",
-                text: translate("fields.address-confirm.change")
+                text: translate("fields.address-confirm.change"),
+                attributes: {
+                  "data-id": "yearFromChange"
```

Comment:
Oh, I was wrong about not being able to target cells directly!

:1st_place_medal:

Reply:
@merlinc We can hit the target cell but we cant hit the value cell 😢